src/test/java/org/sagebionetworks/bridge/spring/controllers/SharedAssessmentControllerTest.java

```diff
@@ -0,0 +1,205 @@
+        when(mockService.importAssessment(TEST_STUDY_IDENTIFIER, OWNER_ID, GUID)).thenReturn(assessment);
+        
+        AssessmentDto retValue = controller.importAssessment(GUID, OWNER_ID);
+        assertEquals(retValue.getGuid(), assessment.getGuid());
+        
+        verify(mockService).importAssessment(TEST_STUDY_IDENTIFIER, OWNER_ID, GUID);
+    }
+    
+    @Test
+    public void getSharedAssessments() {
+        PagedResourceList<Assessment> page = new PagedResourceList<>(ImmutableList.of(mockAssessment), 100);
+        when(mockService.getAssessments(
+                SHARED_STUDY_ID_STRING, 10, 25, STRING_CATEGORIES, STRING_TAGS, true)).thenReturn(page);
+        
+        PagedResourceList<AssessmentDto> retValue = controller.getSharedAssessments("10", "25", STRING_CATEGORIES, STRING_TAGS, "true");
+        assertEquals(retValue.getItems().size(), 1);
+        assertEquals(retValue.getTotal(), Integer.valueOf(100));
+        
+        verify(mockService).getAssessments(
+                SHARED_STUDY_ID_STRING, 10, 25, STRING_CATEGORIES, STRING_TAGS, true);
+    }
+    
+    @Test
+    public void getSharedAssessmentsNoArguments() {
+        PagedResourceList<Assessment> page = new PagedResourceList<>(ImmutableList.of(), 0);
+        when(mockService.getAssessments(
+                SHARED_STUDY_ID_STRING, 0, API_DEFAULT_PAGE_SIZE, null, null, false)).thenReturn(page);
+        
+        PagedResourceList<AssessmentDto> retValue = controller.getSharedAssessments(null, null, null, null, "false");
+        // Just verify this was returned
+        assertEquals(retValue.getTotal(), Integer.valueOf(0));
+        
+        verify(mockService).getAssessments(
+                SHARED_STUDY_ID_STRING, 0, API_DEFAULT_PAGE_SIZE, null, null, false);
+    }
+    
+    @Test
+    public void getSharedAssessmentByGuid() {
+        Assessment assessment = AssessmentTest.createAssessment();
+        when(mockService.getAssessmentByGuid(SHARED_STUDY_ID_STRING, GUID)).thenReturn(assessment);
+        
+        AssessmentDto dto = controller.getSharedAssessmentByGuid(GUID);
+        assertEquals(dto.getGuid(), GUID);
+    }
+    
+    @Test
+    public void getLatestSharedAssessment() {
+        Assessment assessment = AssessmentTest.createAssessment();
+        when(mockService.getLatestAssessment(SHARED_STUDY_ID_STRING, IDENTIFIER)).thenReturn(assessment);
+        
+        AssessmentDto dto = controller.getLatestSharedAssessment(IDENTIFIER);
+        assertEquals(dto.getIdentifier(), IDENTIFIER);
+    }
+
+    @Test
+    public void getSharedAssessmentById() {
+        Assessment assessment = AssessmentTest.createAssessment();
+        when(mockService.getAssessmentById(SHARED_STUDY_ID_STRING, IDENTIFIER, 10)).thenReturn(assessment);
+        
+        AssessmentDto dto = controller.getSharedAssessmentById(IDENTIFIER, "10");
+        assertEquals(dto.getGuid(), GUID);
+    }
+
+    @Test
+    public void getSharedAssessmentRevisionsByGuid() {
```

Comment:
Add test for default values for offset, pageSize, and includeDeleted?

Reply:
Added